client/src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,42 @@
+import useStyles from './useStyles';
+import { useAuth } from '../../context/useAuthContext';
+import { AppBar, Toolbar, Button, IconButton, Link, Box } from '@material-ui/core';
+import AvatarDisplay from '../AvatarDisplay/AvatarDisplay';
+import { Link as routerLink } from 'react-router-dom';
+import NotificationsMenu from './NotificationsMenu/NotificationsMenu';
+export default function NavBar(): JSX.Element {
+  const classes = useStyles();
+  const { loggedInUser } = useAuth();
+
+  return (
+    <div className={classes.root}>
+      {loggedInUser && (
```

Comment:
So, at some point, we will want to update this to conditionally show links based on account type. Sitters and users who needed their dog watched would need different contexts.